pkg/graph/schema/types/user_view_customization.graphql

```diff
@@ -0,0 +1,35 @@
+enum ViewCustomizationType {
+  MY_MODEL_PLANS
+  ALL_MODEL_PLANS
+  FOLLOWED_MODELS
+  MODELS_WITH_CR_TDL
+  MODELS_BY_OPERATIONAL_SOLUTION
+}
+
+type UserViewCustomization {
+  id: UUID!
+  userAccount: UserAccount!

```

Comment:
I'd removed it to just make the GQL type cleaner, since I didn't expect a need for that field specifically (especially with the userAccount type) but I'll add it back for the sake of parity

Reply:
98780938